kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -122,7 +122,8 @@ private void updateStubs() {
   }
 
   private KaldbServiceGrpc.KaldbServiceFutureStub getKaldbServiceGrpcClient(String server) {
-    return Clients.newClient(server, KaldbServiceGrpc.KaldbServiceFutureStub.class)
+    return GrpcClients.builder(server)
+        .build(KaldbServiceGrpc.KaldbServiceFutureStub.class)
```

Comment:
Add a comment stating that this is a client side compression and we need to set the server side separately.

Reply:
Done.